src/05_lists.py

```diff
@@ -8,22 +8,43 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+
+# x.append(int(4))
+
+# or
+
+x.insert(3,int(4))
+
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+y=[8,9,10]
+x.extend(y)
+
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE
+
+x.pop(4)
+
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
 # YOUR CODE HERE
+
+x.insert(5,99)
+
 print(x)
 
 # Print the length of list x
 # YOUR CODE HERE
 
+print(len(x))
+
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+for item in x:
+    print(item)
```

Comment:
this does print each item but is not printing the value they are looking for.
what would you have to do here to print each item times 1000?

Reply:
I missed the x1000 instruction 🤦‍♂️

print((item*1000))